### PR TITLE
Title icons

### PR DESCRIPTION
**Pull request by colegoldsmith**
The idea here is to allow a page attribute to set an icon that gets placed next to the title, most notably to add proper branding for integrations pages

Example:
`:page-icon: ROOT:ui/icons/langchain.svg`

Screenshot 2023-11-13 at 2 46 13 PM

### CODE REVIEW

**Comment by mlr** [comment]
UI bundle preview build successful! :white_check_mark:
Deploying preview to GitHub Pages.

**Comment by mlr** [comment]
Antora site build successful! :white_check_mark:
Deploying draft to GitHub Pages.

File: src/partials/tutorial-info.hbs [hunk 1]
```diff
@@ -19,8 +19,8 @@
   {{/with}}
   <div class="tutorial-actions">
     {{#with page.attributes.colab-link}}
-    <a href="{{{this}}}" target="_blank" class="colab btn btn-outlined btn-neutral">
-      <div class="mr-1">{{{svg 'colab'}}}</div>
```
Comment (colegoldsmith) [inline src/partials/tutorial-info.hbs:23]: I realized the previous svg helper wasn't that useful since we can just use css to set this content.

**Review by mlr — APPROVED** [review]

**Comment by mlr** [comment]
Antora site build successful! :white_check_mark:
Deploying draft to GitHub Pages.

**Pull request merged**